packages/unminify/src/transformations/__tests__/un-jsx.spec.ts

```diff
@@ -154,16 +154,15 @@ inlineTest('jsx with dynamic Component tag',
   `
 function fn() {
   return React.createElement(
-    r ? "a" : "div",
```

Comment:
This test case shouldn't be changed. This is for "dynamic" Component tags.

Reply:
If we are talking about, I should initialise the `r` then I think I have added this test case -> https://github.com/pionxzh/wakaru/pull/129/files#r1590658669